Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -22,6 +22,17 @@ namespace DSP
 {
 namespace HLE
 {
+static bool HasLpf(u32 crc)
+{
+  switch (crc)
+  {
+  case 0x4e8a8b21:
```

Comment:
UCode variations should be handed by each individual UCode, imo.

See the [Zelda UCode](https://github.com/dolphin-emu/dolphin/blob/7fa14169e9aafd011cbb658e32a259e701812be0/Source/Core/Core/HW/DSPHLE/UCodes/Zelda.cpp#L75) for an example. I feel this matches the intended usecase for the m_crc member.